### PR TITLE
[FSK-84]: Document Environment Variables

### PR DESCRIPTION
**Pull request by ASafaeirad**
This PR adds documentation for environment variables

### CODE REVIEW

**Review by pourdaavar — APPROVED** [review]

**Review by MohammadForoutan — APPROVED** [review]

File: .example.env [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
 HOST=
+
+# {String} - a Postgress database URL
 DATABASE_URL=
+
+# {Boolean} - Allow introspection GraphQL server
 INTROSPECTION_ENABLED=
+
+# {String} - a Postgress database URL to use as a shadow database for prisma migration
+SHADOW_DATABASE_URL=
```
Comment (mhmda-83) [inline .example.env]: you need to add this to src/configuration/validation.ts also and src/configurations/EnvironmentVariables.ts

**Review by mhmda-83 — CHANGES_REQUESTED** [review]

**Comment by mhmda-83** [comment]
"Important: The shadow database is not required in production, and is not used by production-focused commands such as prisma migrate resolve and prisma migrate deploy."

Aren't we using migrate deploy for production?
And for development environment we are using local postgresql.

**Review by MohammadForoutan — APPROVED** [review]

File: .example.env (hunk 1 above)
Comment (ASafaeirad) [reply .example.env]: We do not use this one in runtime environment.

**Comment by ASafaeirad** [comment]
> "Important: The shadow database is not required in production, and is not used by production-focused commands such as prisma migrate resolve and prisma migrate deploy."
> 
> Aren't we using migrate deploy for production? And for development environment we are using local postgresql.


If we're going to use local DB it's not a problem. but should we do that?

**Review by pourdaavar — APPROVED** [review]

**Comment by mhmda-83** [comment]
> > "Important: The shadow database is not required in production, and is not used by production-focused commands such as prisma migrate resolve and prisma migrate deploy."
> > Aren't we using migrate deploy for production? And for development environment we are using local postgresql.
> 
> If we're going to use local DB it's not a problem. but should we do that?

why not?

**Pull request merged**